build.sbt

```diff
@@ -1197,6 +1197,7 @@ lazy val developmentTestsDeploymentManager = (project in development("deployment
     name := "nussknacker-development-tests-manager",
   ).dependsOn(
   deploymentManagerApi % "provided",
+  flinkDeploymentManager % "provided",
```

Comment:
`provided` will be enough? Then it will require to use it allways with  `flinkDeploymentManager` in runtime

Reply:
When I declared it with test scope, the DevelopmentDeploymentManagerProvider, which is in main (not test) folder could not access it. But I'm thinking of reverting these introduced dependencies for tests so that testing deployment managers don't have to rely on the flink deployment manager.